game.py

```diff
@@ -12,7 +12,17 @@ class Random(Bot):
     """
     Cory's bot :)
     """
-    pass
+
+    def move(self, state):
+        valid_pile_indexes = []
+        for i, pile in enumerate(state):
+            if pile >=1:
+                valid_pile_indexes.append(i)
+        if not valid_pile_indexes:
+            raise NimException(f'No valid moves on state: {state}')
```

Comment:
Nice use of f-strings! This is actually my first time seeing it being used in an actual project :heart_eyes:

Reply:
Good job Jesse! =p